app/services/meeting_starter.rb

```diff
@@ -17,7 +17,9 @@ def call
 
     retries = 0
     begin
-      BigBlueButtonApi.new.start_meeting room: @room, options: options, presentation_url: @presentation_url
+      meeting = BigBlueButtonApi.new.start_meeting room: @room, options: options, presentation_url: @presentation_url
+
+      @room.update!(last_session: Time.zone.at(meeting[:createTime] / 1000)) unless meeting[:messageKey] == 'duplicateWarning'
```

Comment:
You don't wanna go that way with `meeting[:messageKey] == 'duplicateWarning'` trust me 😆.
It's safer to check for possible mismatches and update the room, `unless @room.last_session == meeting_create_time`.
[MORE](https://github.com/bigbluebutton/greenlight/pull/3000).

Reply:
I don't usually take stuff from v2, but I thought this one made sense and was fail proof.

Reply:
Yeah, you got reason I'd do that way myself.
That one caused us some headache when working on `v2` 🤕